www/src/utils/colors.js

```diff
@@ -4,6 +4,7 @@ const colors = {
   // original palette by @SachaG
   // @see https://www.figma.com/file/J6IYJEtdRmwJQOrcZ2DfvxDD/Gatsby
   gatsby: `#663399`, // was #744c9e
+  gatsbyDark: `#442266`,
```

Comment:
were these color additions discussed with @fk? just making sure.

Reply:
@amberleyromo No, sorry I forgot, I got it from Figma

Reply:
And did not want to add it as a hard coded inline value inside the component, but definitely @fk sould take a loot at this.

Reply:
👍 All fine (as expected ;-))—also not explicitly discussing those before IMO as the design defines them.
Actually I saw them a couple days already and was happy to find out I managed to cook up a design that doesn't add 10 greys ;-) …

That said, I absolutely will appreciate a heads up if I go overboard on new colors (or too light ones, I do that way too often), or any critique or suggestion on how to keep color additions down to a minimum. More generally speaking—in any case, if there's a better way to archive a similar goal, or you have any concerns about what I'm suggesting, or better ideas: Please don't hesitate to bring it up!